tools/mmp/driver.cs

```diff
@@ -408,9 +408,20 @@ static void Main2 (string [] args)
 			if (!targetFramework.HasValue)
 				targetFramework = TargetFramework.Default;
 
-			if (TargetFramework.Identifier == TargetFramework.Xamarin_Mac_2_0.Identifier) {
```

Comment:
The Identifier is Xamarin.Mac, which is in common with Xamarin_Mac_4_5_Full and Xamarin_Mac_4_5_System as well - 🙀